documentation/reference/qfieldcloud/jobs.md

```diff
@@ -0,0 +1,115 @@
+
+!!! warning
+    All jobs must finished within 10 minutes or they will result in timeout error and will be terminated.
+
+## Job types
+
+### Process project file (`process_projectfile`) job
+
+The process project file job is used to extract details about the project configuration and project layers, such as project CRS, layer CRS, layer name, layer validity etc. QFieldCloud validates the uploaded QGIS project file (`.qgs`/`.qgz`), as well as the supporting GeoPackages, TIFFs and other data source files. It also validates remote connection to PostGIS, WFS, WMS and other online data sources. QFieldCloud will open the project file in a QGIS instance on the server to extract all the necessary information.
+
+#### Triggers
+
+This job is triggered every time a file is uploaded to QFieldCloud, unless at least one of the following condition are valid:
+
+- No QGIS project file (`.qgs`/`.qgz`) has been uploaded yet.
+- The uploaded file is within the `DCIM` directory. Those files are assumed to be irrelevant to project validity.
+- There is already a `process_projectfile` job in `PENDING` status.
+
+#### Troubleshoot
+
+A `process_projectfile` job might result in `FAILED` status. Check the non-exhaustive list of causes bellow:
+
+- The uploaded QGIS project file (`.qgs`/`.qgz`) is unreadable, incomplete, broken or wrong. Try to reupload the QGIS project file.
+- QGIS is crashing after opening the project file. Try to identify the layer that is causing the crash by removing one layer at time from the project and reuploading the QGIS project file.
+
+!!! NOTE
+    Even if a `process_projectfile` job results in a `SUCCESS` status, it does not mean the project is properly configured. The `SUCCESS` status just states the project has been successfully opened and all the needed information has been extracted.
+
+### Package (`package`) job
+
+The `package` job convert a QGIS project to a QField project, the same way it is done on QGIS via QFieldSync. The `package` job will prepare all layers marked as "Offline editing" to a single GeoPackage.
+
+#### Triggers
+
+This job is triggered every time the **Download** or **Synchronize** buttons are pressed on QField. Unless at least one of the following condition are valid:
+
+- The project never run a `process_projectfile` job that resulted in `SUCCESS` status.
+- There is already a `package` job in `PENDING` status.
+- The project contains does not contain online vector layers (PostGIS, WFS etc), the latest `package` job result was `SUCCESS` and there were not file uploads, nor change uploads.
+
+#### Troubleshoot
+
+A `package` job might result in `FAILED` status. Check the non-exhaustive list of causes bellow:
+
+- The project never run a `process_projectfile` job that resulted in `SUCCESS` status.
+- Some of the project layers are inaccessible from QFieldCloud. Make sure all files are uploaded and all credentials to online layers (PostGIS, WFS etc) are stored within the QGIS project file.
```

Comment:
For this section, would it be good to give a hint to the job log that could contain more information about the failure reason?

Reply:
I mentioned this in the begining and after that I have a section "Troubleshoot job logs" which should be enough. It would be great if we add a few screenshots here and there, will make the life of the reader easier too.